drivers/watchdog/wdt_mcux_wwdt.c

```diff
@@ -83,7 +83,7 @@ static int mcux_wwdt_install_timeout(const struct device *dev,
 	}
 
 #if defined(CONFIG_SOC_MIMXRT685S_CM33) || defined(CONFIG_SOC_MIMXRT595S_CM33) \
-	|| defined(CONFIG_SOC_SERIES_MCXNX4X)
+	|| defined(CONFIG_SOC_SERIES_MCXNX4X) || defined(CONFIG_SOC_SERIES_MCXN23X)
```

Comment:
when you change the Kconfig, you'll have to put this change in the soc commit to retain bisectibility (and change any other occurrences)